examples/filters.py

```diff
@@ -0,0 +1,172 @@
+# ---
+# jupyter:
+#   jupytext:
+#     formats: ipynb,py
+#     text_representation:
+#       extension: .py
+#       format_name: light
+#       format_version: '1.5'
+#       jupytext_version: 1.8.0
+#   kernelspec:
+#     display_name: Python 3
+#     language: python
+#     name: python3
+# ---
+
+# # Filter Examples
```

Comment:
@jorshi Last suggestion. Let's put ALL the examples in one `examples.py` file, maybe with different sections.

Then it's easy to run code-coverage, which usually is easiest when you call only one file (`examples.py` in this case).